sztp-agent/pkg/secureagent/utils.go

```diff
@@ -159,29 +160,44 @@ func generateInputJSONContent() string {
 
 type publicKey struct {
```

Comment:
can we remove this struct and use https://pkg.go.dev/golang.org/x/crypto/ssh#PublicKey directly ?